### ISSUE TITLE
NOWEB engine with Profile picture 

### ISSUE THREAD

**Issue by Khosravangroup** [allburov/whatsapp-http-api#298]
Hello guys

i think with NOWEB Engine needs to get profile picture 

please add this feature to NOWEB..

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
I'll look on that on this weekend!

**Comment by Khosravangroup** [comment]
Thanks , please add this feature..

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Hi! Fixed in **2024.4.4**, make sure to update the image and restart the container

https://waha.devlike.pro/docs/how-to/contacts/#get-contact-profile-picture

----

### Get contact profile picture

- `GET /api/contacts/profile-picture?contactId=11231231231&session=default`

```json
{
  "profilePictureURL": "https://example.com/profile.jpg"
}
```
